src/components/Avatar/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { View,Text } from 'react-native';
+import { Loading } from '../Loading';
+import { useAuthStore } from '../../shared/zustand/auth';
+import { THEME_COLORS, THEME_FONTS } from '../theme';
+import { styles } from './style';
+
+
+const Avartar = ({
+    type = "circle",
+    avatarUri,
+    width,
+    height,
+    content,
+}) => {
+    const { isLoading } = useAuthStore();
+    if (isLoading) {
+        return <Loading />;
+    }
+    if (type == "square") {
+        return (
```

Comment:
You can simplify your code. 
Try to avoid as many duplicates as you can.
e.g., change to 
```

const avatarUniqueStyle = React.useMemo(()=>{
switch(type){
  case 'square':
      return {
         width, height,
         backgroundColor: THEME_COLORS.DEFAULT_BLUE_PRIMARY
      }
  case 'roundReact':
      return ....
  default:
      return ....
}
                    
},[width, height, type])

return (
            <View
                style={[
                    styles.avatarContainer,
                    avatarUniqueStyle
                ]}
            >
                <Text style={styles.text}>{content}</Text>
            </View>
        );
```